code/__DEFINES/jobs.dm

```diff
@@ -175,3 +178,56 @@
+#define JOB_GARRISON_ROLES	/datum/timelock/garrison
+#define JOB_INQUISITION_ROLES	/datum/timelock/inquisition
+#define JOB_NOBLE_ROLES	/datum/timelock/noble
+#define JOB_MERCHANT_COMPANY_ROLES	/datum/timelock/merchant_company
+#define JOB_ADVENTURER_ROLES	/datum/timelock/adventurer
+#define JOB_LEADERSHIP_ROLES	/datum/timelock/leadership
+#define JOB_MEDICAL_ROLES	/datum/timelock/medical
+#define JOB_MAGICK_ROLES	/datum/timelock/magick
+#define JOB_ARTIFICER_ROLES	/datum/timelock/artificer
+#define JOB_THIEF_ROLES	/datum/timelock/thief
+#define JOB_BARD_ROLES	/datum/timelock/bard
+#define JOB_RANGER_ROLES	/datum/timelock/ranger
+#define JOB_MERCENARY_ROLES	/datum/timelock/mercenary
+
+/// The only reason some jobs with advclasses also have their job datum is because in the past advclass were not jobs and it didn't saved in the DB, for the NEW jobs that got advclasses, ONLY put the advclasses datums.
+
+#define JOB_CHURCH_ROLES_LIST	list(/datum/job/priest, /datum/job/monk, /datum/job/templar, /datum/job/undertaker)
+#define JOB_GARRISON_ROLES_LIST	list(/datum/job/dungeoneer, /datum/job/forestguard, /datum/job/advclass/forestguard/infantry, /datum/job/advclass/forestguard/ranger, /datum/job/advclass/forestguard/reaver, /datum/job/advclass/forestguard/brawler, /datum/job/advclass/forestguard/ruffian, /datum/job/forestwarden, /datum/job/guardsman, /datum/job/advclass/garrison/footman, /datum/job/advclass/garrison/archer, /datum/job/advclass/garrison/pikeman, /datum/job/gatemaster, /datum/job/advclass/gatemaster/gatemaster_whip, /datum/job/advclass/gatemaster/gatemaster_mace, /datum/job/advclass/gatemaster/gatemaster_bow, /datum/job/jailor, /datum/job/men_at_arms, /datum/job/advclass/menatarms/watchman_pikeman, /datum/job/advclass/menatarms/watchman_swordsmen, /datum/job/advclass/menatarms/watchman_ranger, /datum/job/royalknight, /datum/job/lieutenant, /datum/job/advclass/royalknight/knight, /datum/job/advclass/royalknight/steam, /datum/job/captain)
+#define JOB_INQUISITION_ROLES_LIST	list(/datum/job/inquisitor,/datum/job/advclass/puritan/inspector, /datum/job/advclass/puritan/ordinator, /datum/job/adept, /datum/job/advclass/adept/bzealot, /datum/job/advclass/adept/rthief, /datum/job/advclass/adept/highwayman, /datum/job/absolver, /datum/job/advclass/confessor, /datum/job/advclass/disciple, /datum/job/advclass/psyaltrist, /datum/job/advclass/psydoniantemplar)
+#define JOB_NOBLE_ROLES_LIST	list(/datum/job/archivist, /datum/job/captain, /datum/job/consort, /datum/job/advclass/consort/highborn, /datum/job/advclass/consort/courtesan, /datum/job/advclass/consort/lowborn, /datum/job/advclass/consort/courtesan/night_spy, /datum/job/courtphys, /datum/job/consort, /datum/job/advclass/hand/hand, /datum/job/advclass/hand/spymaster, /datum/job/advclass/hand/advisor, /datum/job/lord, /datum/job/magician, /datum/job/minor_noble, /datum/job/steward)
+#define JOB_MERCHANT_COMPANY_ROLES_LIST	list(/datum/job/merchant, /datum/job/grabber, /datum/job/shophand)
```

Comment:
add travelling merchant.

Reply:
wait travelling merchant don't means you actually know how to do merchant stuff, you are just a pilgrim from the company